### PR TITLE
LA-349 Authentication app with biometric

### PR DESCRIPTION
**Pull request by dab246**
**Demo Android Device**

https://user-images.githubusercontent.com/80730648/116525914-3854a700-a903-11eb-97a6-f727080e312e.mp4

**Demo iOS Device : iPhone XR**

https://user-images.githubusercontent.com/80730648/116540128-ee27f180-a913-11eb-9110-c4e9333d4f0e.mov

### CODE REVIEW

File: domain/lib/src/model/biometric_authentication/ios_setting_arguments.dart [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+//  3 and <http://www.linshare.org/licenses/LinShare-License_AfferoGPL-v3.pdf> for
+//  the Additional Terms applicable to LinShare software.
+
+import 'package:domain/src/model/biometric_authentication/platform_setting_arguments.dart';
+
+class IOSSettingArgument extends PlatformSettingArguments {
+  final String cancelButton;
+
+  IOSSettingArgument(this.cancelButton);
+}
```
Comment (hoangdat) [inline domain/lib/src/model/biometric_authentication/ios_setting_arguments.dart:38]: Can not understand

Comment (dab246) [reply domain/lib/src/model/biometric_authentication/ios_setting_arguments.dart:38]: I created a parameter object for different platforms to edit the biometric authentication dialog interface

**Comment by imGok** [comment]
Just a question about the fingerprint location, is it constant or based on the user device ?

**Comment by dab246** [comment]
> Just a question about the fingerprint location, is it constant or based on the user device ?

Fingerprints based on the user device.

**Review by hoangdat — APPROVED** [review]

**Pull request merged**